java/oauth-custom-challengehandler/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        tools:context=".MainActivity">
+
+    <com.esri.arcgisruntime.mapping.view.MapView android:id="@+id/mapViewLayout"
```

Comment:
I would name the id `mapView` instead of `mapViewLayout`.

Reply:
updated